api/app/Policies/UserPolicy.php

```diff
@@ -55,7 +55,7 @@ public function viewBasicInfo(User $user)
      */
     public function create(User $user)
     {
-        return $user->isAdmin(); //TODO: return false when createUser mutation is removed.
+        return $user->isAdmin();
```

Comment:
My thought was that the `isAdmin` function and all the references to it would be changed as part of https://github.com/GCTC-NTGC/gc-digital-talent/issues/5440. If my thought is incorrect, let me know and I will work on creating an appropriate permission and changing it here.

Reply:
In scrum: add new permission and start using it in this function as part of this PR.

Reply:
See: 

- ec69edcd3e8790c1a8ad37b770f9a7954bc7f459
- 9b5541aed73ea4d2af2d5e0b53575469f16eb7d4

Also, added new permission to row 31: https://docs.google.com/spreadsheets/d/1RC7wufxwjLEvaEunHsfZfOzBefg218dvvzkDwKSiQjI/edit#gid=428263758

Reply:
Nice, testing now.  Could you add the deployment tag to the issue and a note in the PR to run the permission seeder in prod?

Reply:
Yep, already done: https://github.com/GCTC-NTGC/gc-digital-talent/issues/6364#:~:text=%F0%9F%9A%80-,Deployment,-Re%2Drun%20RolePermissionSeeder